lib/services/local/presence.go

```diff
@@ -610,18 +610,37 @@ func (s *PresenceService) CreateRemoteCluster(rc services.RemoteCluster) error {
 	return nil
 }
 
-// UpdateRemoteCluster updates remote cluster
+// UpdateRemoteCluster updates selected remote cluster fields: expiry and labels
+// other changed fields will be ignored by the method
```

Comment:
why not update all the fields?
this seems like maintenance burden

Reply:
to avoid updating parts that should not be updated - like connection status or last heartbeat

Reply:
How about overwriting those fields with the current backend values then?
Instead of copying over just the fields that can be updated, suppress the changes to the fields that can't be updated.